app/javascript/components/layout/CustomTable.js

```diff
@@ -235,35 +235,39 @@ class CustomTable extends React.Component {
               </Col>
               <span className="ml-2 text-nowrap align-self-center">{`Displaying ${this.props.rowData.length} out of ${this.props.totalRows} rows.`}</span>
```

Comment:
This could also wrap under the show dropdown so I would play around with the bootstrap breakpoints a little more